scripts/linux.sh

```
@@ -26,6 +26,13 @@ for disk in $DISKS; do
 
   device="/dev/$disk"
   diskinfo=($(lsblk -b -d "$device" --output SIZE,RO,RM,MODEL | ignore_first_line))
```

It already get's silently swallowed :)